docs/dev/framework/asset-management.md

```diff
@@ -0,0 +1,205 @@
+  query parameter to the file for cache busting. The latter can also be set to
+  `null` in order to automatically use the file's modification time for cache busting
+  (if it is a relative file path).
+
+  ```php
+  $GLOBALS['TL_HEAD'][] = \Contao\Template::generateStyleTag('bundles/myextension/print.css', 'print', null);
+  ```
+* __`\Contao\Template::generateInlineStyle($script)`__
+
+  This wraps the given CSS with `<style>…</style>`.
+
+  ```php
+  $GLOBALS['TL_HEAD'][] = \Contao\Template::generateInlineStyle($this->generateCss());
+  ```
+* __`\Contao\Template::generateScriptTag($href, $async, $mtime, $hash, $crossorigin, $referrerpolicy)`__ 
+  
+  This returns a `<script src="…" …>` tag and takes six arguments: 
+
+  * `$href`: the path to the stylesheet (absolute or relative to the _base_)
+  * `$async`: whether the `async` attribute should be added to the tag (default `false`)
+  * `$mtime`: an optional modification time, which Contao will use to append a query 
+    parameter to the file for cache busting. This can also be set to `null` in order 
+    to automatically use the file's modification time for cache busting.
+  * `$hash`: optional hash for an `integrity` attribute.
+  * `$crossorigin`: optional `crossorigin` attribute.
+  * `$referrerpolicy`: optional `referrerpolicy` attribute.
+
+  {{% notice note %}}
+Some of these parameters are only available in newer Contao versions.
+  {{% /notice %}}
+
+  ```php
+  $GLOBALS['TL_BODY'][] = \Contao\Template::generateScriptTag('bundles/myextension/scripts.js', false, null);
+  ```
+* __`\Contao\Template::generateInlineScript($script)`__
+
+  This wraps the given JavaScript with `<script>…</script>`.
+
+  ```php
+  $GLOBALS['TL_BODY'][] = \Contao\Template::generateInlineScript($this->generateJavaScript());
+  ```
+* __`\Contao\Template::generateFeedTag($href, $format, $title)`__
+
+  This generates a `<link type="application/…" rel="alternate" href="…" title="…">` 
+  tag for RSS feeds. It takes three arguments: the URL to the feed, its format and 
+  the title of the feed.
+
+  ```php
+  $GLOBALS['TL_HEAD'][] = \Contao\Template::generateFeedTag('share/myfeed.xml', 'rss', 'My Feed');
+  ```
+
+
+## Accessing Assets in Templates
+
+{{< version "4.5" >}}
+
+Within [Contao Templates][ContaoTemplates] you can access the helper function `$this->asset(…)`
+in order to automatically link to specific assets of either a "Contao Component"
+or a bundle. It takes two arguments: the path of the asset within the bundle or
+component, and the bundle or component name.
+
+For example, in order to integrate the tablesorter component (which Contao installs
+as a dependency through `contao-components/tablesort`) within a template, the following
+can be used:
+
+```php
+<script src="<?= $this->asset('js/tablesort.min.js', 'contao-components/tablesort') ?>"></script>
+```
+
+The same can be used for assets of a bundle. Suppose you have a package called `example/contao-foo-extension`
```

Comment:
I'm not sure this is correct. If I remember correctly, bundles use the bundle name, not the package name, as each package can contain multiple bundles. We only use the package name for `contao-component`s. Let me know if you need help digging that in the Contao code 🙈

Reply:
To be honest, I did not test this my self 😁 . But the parameter is called `$packageName`, not `$bundleName`, and for the Contao Components, you also use the package name. But looking at `AddAssetsPackagesPass::addBundles`, it seems to be indeed the bundle name.